src/jit/compiler.cpp

```diff
@@ -4835,6 +4835,10 @@ void Compiler::compCompile(void** methodCodePtr, ULONG* methodCodeSize, JitFlags
 #endif
 
 #if defined(DEBUG)
+    if (JitConfig.DumpJittedMethods() == 1)
+    {
+        printf("  Generated 0x%x bytes at 0x%p\n", (unsigned)(*methodCodeSize), *methodCodePtr);
```

Comment:
I'm not sure how others will like this. For me, I frequently add this code when I'm debugging, so it seemed useful to keep it. But it doubles the number of lines dumped for the case when you just want to know what's compiled and/or what the method hashes are.

Reply:
I always use `COMPlus_JitFunctionTrace` which does this already. Actually, I wonder if we need DumpJittedMethods.

Reply:
I can't believe I never knew about that. I'm not crazy about the format but it basically does what it needs. To me it's weird that it dumps the method index as hex, and that it calls them "Entry" rather than "Method". Also I'm not sure what the point of the `FMT_ADDR` which dumps the address in a weird format (e.g. `00000195`567ad060`) that can't easily be copied and pasted. Do you know if there's some fundamental reason for that?

Reply:
I don't think there's any particular format required for JitFunctionTrace. The FMT_ADDR is just to be consistent with other places in the runtime, and is what windbg does for 64-bit addresses, but it is sometimes annoying to copy/paste if the target of the paste doesn't support reading that format. Doesn't matter to me if you change it.